src/repository/StudentRepo.java

```diff
@@ -0,0 +1,19 @@
+package repository;
+
+import model.Student;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public interface StudentRepo {
+    static List<Student> getAll(){
```

Comment:
Do not use static method here. I want you to create new instance of repository in StudentService when you call method of StudentRepository